### DISCUSSION TITLE
Move the table filter next to the title

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#14454] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

Hi.

Is it possible to move the table filter up to the title header, so it doesn't consume so much space?

![image](https://github.com/user-attachments/assets/800fb033-be70-4177-b630-397e4406a982)

**Comment by leandrocfe** [comment]
You can do this..

https://filamentphp.com/docs/3.x/tables/filters/layout/#displaying-filters-above-the-table-content

**Comment by oliverbj** [comment]
Thanks - my purpose was to save some screen estate, and thus avoid the second bar (since it only consists of a filter icon).
